website/versioned_docs/version-0.24.0/enterprise/getting-started/aws-marketplace.mdx

```diff
@@ -203,4 +202,4 @@ Your Weave GitOps installation is now ready!
 
 ## Next steps
 
-In our following [Get Started document](../getting-started/ui.mdx), we will walk you through logging into the GitOps Dashboard and deploying an application.
+In our Get Started document, we will walk you through logging into the GitOps Dashboard and deploying an application.
```

Comment:
I guess that would be better to have a link to the getting started or to remove the section
<img width="1093" alt="Screenshot 2023-09-08 at 15 10 45" src="https://github.com/weaveworks/weave-gitops/assets/12957664/9db7e383-aa5b-4776-b1f4-346e3e30a336">

Reply:
@enekofb I didn't edit the docs content and don't see the value of updating these old versions.

Reply:
Removed section